README.md

```diff
@@ -45,6 +46,10 @@ Specify a fallback page for SPA mode, e.g. `index.html` or `200.html` or `404.ht
 
 If `true`, precompresses files with brotli and gzip. This will generate `.br` and `.gz` files.
 
+### manifest
+
+Specify different files for different target platforms, e.g. `chrome_manifest.json`, `firefox_manifest.json`
```

Comment:
Could you add the following information: 
"This file name must match one that is present in the `static` directory (the dir containing your static files). The selected target file will be renamed to `manifest.json` in the build directory"